src/compile/legend/assemble.ts

```diff
@@ -16,8 +16,8 @@ function setLegendEncode(
   legend.encode ??= {};
   legend.encode[part] ??= {};
   legend.encode[part].update ??= {};
-  // TODO: remove as any after https://github.com/prisma/nexus-prisma/issues/291
-  (legend.encode[part].update[vgProp] as any) = vgRef;
+  // @ts-expect-error expression is too complex for typescript to understand
```

Comment:
`expression is too complex for typescript to understand`

Confirming understanding that this  PR was merged, but the issue still appears.